holidayextras/holidayextras.github.com
# Fixes links in RSS feed

## What does this PR do?
URL was missing in config so RSS links didn't work

## How to test
- Checkout branch
- Run `jekyll serve`
- Visit `http://localhost:4000/feed.xml`
- All links should start `http://holidayextras.github.com`

- Run `jekyll serve --config _config.yml,_config_dev.yml`
- Visit `http://localhost:4000/feed.xml`
- All links should start `http://localhost:4000`

## Give me a GIF
![](http://i.giphy.com/93nQo4zR3j1OE.gif)
![](http://i.giphy.com/52ysheIwJTRS0.gif)

## Review

#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
